README.md

```diff
@@ -29,15 +29,15 @@ Add the following dependency to your pom.xml:
 ### Using with sbt
 
 ```
-libraryDependencies += "com.intel.chimera" % "chimera" % "0.9.0"
+libraryDependencies += "org.apache.commons.crypto" % "chimera" % "0.9.0"
```

Comment:
Please note there is still a "chimera" in the name part. If this change for only package name, please describe this specifically in the JIRA. 
Make a note for this instance so that we remember change it in the future.